libnmstate/nm/plugin.py

```diff
@@ -140,6 +144,18 @@ def get_dns_client_config(self):
     def refresh_content(self):
         self._ctx.refresh_content()
 
+    @staticmethod
+    def check_version_mismatch():
+        nm_client_version = _nm_client_decode_version()
```

Comment:
This should be `nm_client_version = _nm_client_decode_version(self._ctx.client())`.

Reply:
My bad. It should be `self._ctx.client`.

Reply:
Done!